src/Mapper/ORM/DoctrineORMMapper.php

```diff
@@ -14,8 +14,8 @@
 namespace Sonata\Doctrine\Mapper\ORM;
 
 use Doctrine\Common\EventSubscriber;
-use Doctrine\Common\Persistence\Event\LoadClassMetadataEventArgs;
 use Doctrine\ORM\Mapping\ClassMetadataInfo;
+use Doctrine\Persistence\Event\LoadClassMetadataEventArgs;
```

Comment:
This one is used in a method signature, you will have to add an autoload call at the bottom of the file, as described in https://dev.to/greg0ire/how-to-deprecate-a-type-in-php-48cf

Reply:
Ah wait nevermind the class is final, my commment does not apply.